packages/react-hooks/README.md

```diff
@@ -187,6 +187,48 @@ const Example = React.forwardRef(function Example(props: {}, forwardedRef: React
 });
 ```
 
+## useMount
+
+```ts
+const useMount: (callback: () => void) => void;
+```
+
+Hook that asynchronously fires a callback before mount.
+
+```tsx
+import { useMount } from '@uifabric/react-hooks';
+
+const MyComponent = () => {
+  useMount(() => {
+    console.log('Example');
+   })
+
+  return <div />;
+};
+});
+```
+
+## useMountSync
+
+```ts
+const useMountSync: (callback: () => void) => void;
+```
+
+Hook that synchronously fires a callback before mount. This should only be used if useMount causes problems as it will
+trigger debug warnings in server-rendered scenarios.
```

Comment:
```suggestion
Hook which synchronously execute a callback when the component has been mounted. Use `useMount` for most scenarios. You should only use the synchronous version in the rare case you need to perform an action after the component has been mounted and before the browser paints, such as measuring content and adjusting the result. Using this will trigger debug warnings in server-rendered scenarios.
```